docs/notes/2.28.x.md

```diff
@@ -46,6 +48,10 @@ Added an option to [the `[docker]` subsystem](https://www.pantsbuild.org/prerele
 bypassing the `suggest_renames` functionality. There are currently situations where this code path can
 [introduce performance degradation](https://github.com/pantsbuild/pants/issues/22246), and this flag allows the end user to work around that issue.
 
+#### Protobuf (Python)
+
+The new `[python-protobuf].generate_type_stubs` option configures `protoc` to directly generate type stubs which has been supported in recent versions of `protoc` for some time. This option should be preferred over the `[python-protobuf].mypy_plugin` option.
```

Comment:
What happens if both are specified?

Reply:
Both options are now defined with `mutally_exclusive_group`. Shouldn't that prevent both of them being enabled?

Reply:
Ah yes, missed that. Excellent.